RandLAPACK/drivers/rl_cqrrp.hh

```diff
@@ -458,33 +459,39 @@ int CQRRP_blocked<T, RNG>::call(
             updating3_t_stop  = high_resolution_clock::now();
             updating3_t_dur  += duration_cast<microseconds>(updating3_t_stop - updating3_t_start).count();
         }
-
-        // Estimate R norm, use Fro norm trick to compute the approximation error
-        // Keep in mind that R11 is Upper triangular and R12 is rectangular.
-        norm_R11 = lapack::lantr(Norm::Fro, Uplo::Upper, Diag::NonUnit, b_sz, b_sz, R11, lda);
-        norm_R12 = lapack::lange(Norm::Fro, b_sz, n - curr_sz - b_sz, R12, lda);
-        norm_R_i = std::hypot(norm_R11, norm_R12);
-        norm_R   = std::hypot(norm_R, norm_R_i);
-        // Updating approximation error
-        approx_err = ((norm_A - norm_R) * (norm_A + norm_R)) / norm_A_sq;
+        /*
+        Termination criteria based on the estimate for the comdition number of a sketch at a given iteration.
+        Does not detect low rank.
+        Is only useful for "bad inputs."
+        */
+       /*
+        if (iter != 0) {
+            //running_max = A_sk[0];
+            running_min = A_sk[0];
+            for(i = 0; i < std::min(sampling_dimension, cols); ++i) {
+                curr_entry = std::abs(A_sk[i * d + i]);
+                running_max = std::max(running_max, curr_entry);
+                running_min = std::min(running_min, curr_entry);
+                if(std::abs(running_min / running_max) <= this->eps) {
+                    printf("%e\n", std::abs(running_min / running_max));
+                    termination_criteria = 1;
+                    break;
+                }
+            }
+        }
+        */
```

Comment:
We keep it if it's useful for bad inputs. Your code should be as reliable as QP3.

Reply:
Same comment as above.